.github/workflows/gh-release.yml

```
@@ -20,4 +20,5 @@ jobs:
         uses: softprops/action-gh-release@v1
         if: startsWith(github.ref, 'refs/tags/')
```

Pourquoi qu'on a plusieurs fois la même condition?  J'ai l'impression que `Generate Changelog-current` pourrait être éxécuter sans même avoir procédé a l'installation précédente?
Mais les conditions sont-elles vraiment nécessaire vu qu'on roule déjà dans un contexte des tags?

Est-ce qu'on devrait retrouver la condition au niveau de la jobs:
```suggestion
        jobs:
  build:
    if: startsWith(github.ref, 'refs/tags/')
    runs-on: ubuntu-latest
    steps:
      - name: Checkout
        uses: actions/checkout@v3
      - name: Install conventional-changelog@3.1.25
        run: npm install conventional-changelog@3.1.25
      - name: Generate Changelog-current
        run: npm run changelog-current |  grep -v "^>" | sed '/^$/d' > ${{ github.workspace }}-CHANGELOG.txt
      - name: Release
        uses: softprops/action-gh-release@v1
        with:
          body_path: ${{ github.workspace }}-CHANGELOG.txt
          token: ${{ secrets.GITHUB_TOKEN }}
```

---

Corrigé.